go/extractor/cli/go-extractor/go-extractor.go

```diff
@@ -21,7 +21,7 @@ func usage() {
 	fmt.Fprintf(os.Stderr, "--help                Print this help.\n")
 }
 
-func parseFlags(args []string, mimic bool) ([]string, []string) {
+func parseFlags(args []string, mimic bool, extractTests bool) ([]string, []string, bool) {
```

Comment:
What is the point of passing this in and then passing it back out again? Because you think in future that it might get used in this function?

Reply:
It gets overridden if we're in mimic (i.e. build-tracing) mode

Reply:
Oh yes, I missed that.

Reply:
Possibly worth documenting here by adding a doc comment for `parseFlags`?